final_prep/annotations.rb

```diff
@@ -16,28 +20,35 @@ def build_a_bear(name, age, fur, clothes, special_power)
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+  #return built bear hash
   return built_bear
 end
-
+#call build_a_bear method for fluffy
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
+#call build_a_bear method for sleepy
 build_a_bear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
 
 
 # FizzBuzz
 
 def fizzbuzz(num_1, num_2, range)
+#for every number between 1 and range...
   (1..range).each do |i|
+    #if i is evenly divisable by num 1 and num 2
```

Comment:
I like this simple explanation for what a modulo evaluates. You may want to discuss this topic with classmates as it is tricky, and good practice in technical explanation.